dub.json

```diff
@@ -8,5 +8,5 @@
     "targetType": "staticLibrary",
     "targetPath": "build",
     "workingDirectory": "build",
-    "preBuildCommands": ["python $PACKAGE_DIR/python/prebuild.py $PACKAGE_DIR"]
+    "preBuildCommands": ["$MATPLOTLIB_D_PYTHON $PACKAGE_DIR/python/prebuild.py $PACKAGE_DIR"]
```

Comment:
I recognize that  "preBuildCommands" is executed when users install this library by using dub. Is this wrong? If it's right, we had better not use any environment variables to specify the version of Python.